serverless/event_sources/knative-event-sources.adoc

```diff
@@ -9,20 +9,20 @@ An _event source_ is an object that links an event producer with an event _sink_
 
 Currently, {ServerlessProductName} supports the following event source types:
 
-ApiServerSource:: Connects a sink to the Kubernetes API server.
-PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
-SinkBinding:: Allows you to connect core Kubernetes resource objects such as a `Deployment`, `Job`, or `StatefulSet` with a sink.
-KafkaSource:: Connect a Kafka cluster to a sink as an event source.
+API server source:: Connects a sink to the Kubernetes API server.
+Ping source:: Periodically sends ping events with a constant payload. It can be used as a timer.
+Sink binding:: Allows you to connect core Kubernetes resource objects, such as `Deployment`, `Job`, or `StatefulSet` objects, with a sink.
+Kafka source:: Connect a Kafka cluster to a sink as an event source.
```

Comment:
It's because here we're talking about the source types as concepts as opposed to referring to the literal CR objects, so for example it would be "ping source" for general use, or "`PingSource` object" if we're specifying something specifically relating to the CR. The guidelines are here if that helps: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
Open Shift :D

...sorry, bad joke